packages/vscode-extension/src/utils/commonUtils.ts

```diff
@@ -135,3 +137,16 @@ export function anonymizeFilePaths(stack?: string): string {
 
   return updatedStack;
 }
+
+export async function isTeamsfx(): Promise<boolean> {
+  if (workspace.workspaceFolders && workspace.workspaceFolders.length > 0) {
+    const workspaceFolder = workspace.workspaceFolders[0];
+    const fxPath = await commonUtils.getProjectRoot(workspaceFolder.uri.fsPath, ".fx");
```

Comment:
Keep consistent to use the const variable instead of hard code here?
```suggestion
    const fxPath = await commonUtils.getProjectRoot(workspaceFolder.uri.fsPath, `.${ConfigFolderName}`);
```

Reply:
looks we can reuse `isFxProject` here like: 

```javascript
return await isFxProject(workspaceFolder.uri.fsPath);
```

Reply:
Fixed.